features/rails_features/integrations.feature

```diff
@@ -7,10 +7,10 @@ Background:
 
 @rails_integrations
 Scenario: Delayed job
-  When I run the "jobs:work" rake task in the rails app
-  And I run "User.new.delay.raise_the_roof" with the rails runner
-  And I wait to receive a request
-  Then the request is valid for the error reporting API version "4.0" for the "Ruby Bugsnag Notifier"
+  When I run "User.new.delay.raise_the_roof" with the rails runner
+  And I run the "jobs:workoff" rake task in the rails app
```

Comment:
The order of these had to change so that the job gets queued first and the job worker runs second, otherwise the job worker command will block the job from being queued in the first place